src/download.jl

```diff
@@ -0,0 +1,49 @@
+"""
+    dataurl(dataset_code)
+
+This function takes in a dataset code of format "GADM/<country_code>"
+Returns the URL of the gpkg dataset of the country
+"""    
+
+function dataurl(dataset_provider, country_code)
```

Comment:
The signature of the function is incorrect. The docstring is correct. Here is the correct logic:

```julia
function dataurl(dataid)
  if contains(dataid, "GADM")
    # produce url for GADM
    provider, id = split(dataid, '/')
    "https://...$id..."
  else
    @error "not implemented"
  end
end
```